components/annot-missing-values.vue

```diff
@@ -91,11 +92,14 @@
         },
 
         methods: {
+            ...mapMutations([
+                "declareNotMissing"
+            ]),
 
-            removeColumn(p_tableItem) {
+            removeValue(tableItem) {
 
                 // Remove this value from the column's missing value list in the store
-                this.$emit('remove:missingValue', p_tableItem);
+                this.declareNotMissing({column: tableItem.column, value: tableItem.value});
```

Comment:
You mean instead of the `this.$emit` or `this.$store`? I think clarity and consistency is best. When we use the mapHelpers anywhere, then I think it's better if we use them everywhere - saves one implicit equivalence someone needs to see / understand to not be confused. 

wdyt?

Reply:
This makes sense to me. I didn't remember the `mapMutations` ability too. Happy to stick to one paradigm.